tests/test_external_account.py

```diff
@@ -275,9 +275,77 @@ def assert_resource_manager_request_kwargs(
         assert request_kwargs["headers"] == headers
         assert "body" not in request_kwargs
 
+    def test_valid_token_url_shall_pass_validation(self):
+        # valid url doesn't throw exception, a None value should be return
```

Comment:
.. # A valid url shouldn't throw an exception and a None value should be returned.

Also fix punctuation everywhere else.